roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -110,6 +110,15 @@ kafka_broker:
   log4j_file: /etc/kafka/kafka_server_log4j.properties
   jaas_file: /etc/kafka/kafka_server_jaas.conf
 kafka_broker_schema_validation_enabled: false
+kafka_broker_metrics:
```

Comment:
what do you think about moving these vars into the respective component's role's default vars file? ie confluent.kafka_broker/defaults/main.yml

Reply:
This can be done